public/partials/io/import.html

```diff
@@ -70,13 +70,38 @@ <h5>Imported data sources</h5>
                 <button ng-click="cancelImport()">cancel</button>
 
             </div>
-            <div class="col-md-4">
-                <h5>Existing data sources</h5>
+            <div class="col-md-3">
+                <h5>Existing Layers</h5>
+                <ul>
+                    <li ng-repeat="item in importFile.layers">
+                        <div>
+                            {{item.name}}
+                        </div>
+                        <input type="checkbox" ng-model="item.replace" ng-value="boolean">
```

Comment:
Nothing indicates what this checkbox is supposed to do. Maybe add a title attribute or a tooltip ?